onedal/_device_offload.py

```diff
@@ -194,17 +187,29 @@ def support_usm_ndarray(freefunc=False, queue_param=True):
 
     def decorator(func):
         def wrapper_impl(obj, *args, **kwargs):
-            usm_iface = _extract_usm_iface(*args, **kwargs)
+            if len(args) == 0 and len(kwargs) == 0:
+                return _run_on_device(func, obj, *args, **kwargs)
```

Comment:
are you sure ```*args``` and ```**kwargs``` need to be passed in this circumstance?

Reply:
Do you think this can break smth?

Reply:
no, but it is a good simplification.